template.tpl

```diff
@@ -116,27 +101,23 @@ setCookie('_km', cid, { 'max-age': 30 * 24 * 60 * 60, 'secure': true });
 // If event is a seedtag.timer then it's a qualified visit, we can record a close.
 let isTimer = copyFromDataLayer('event') === 'seedtag.timer';
 
+// Optional eventData could be set in data layer
+let eventData = copyFromDataLayer('eventData');
+
 var sUrl = 'https://t.kmtx.io/s?' +
-    'aid=' + encodeUriComponent(data.aid) +
+    'pid=' + encodeUriComponent(data.pid) +
     '&cid=' + cid +
     '&eid=' + uuid() +
     '&a=' + (isTimer ? 'close' : data.type) +
-    '&v=gtm_1' +
+    '&ed=' + (eventData === undefined ? '' : encodeUriComponent(eventData)) +
+    '&v=gtm_2' +
```

Comment:
should we use `gtm_3.0` to match the new kmpx?
but then should we call this `kmpx_3.0` in the `kmpx.js`?

Reply:
it is designed to be a float in kmpx.js

Reply:
I will put gtm_3.0